tests/upgrades/test_syncplan.py

```diff
@@ -15,33 +15,22 @@
 :Upstream: No
 """
 from fauxfactory import gen_choice
-from fauxfactory import gen_string
 from nailgun import entities
-from requests.exceptions import HTTPError
 from upgrade_tests import post_upgrade
 from upgrade_tests import pre_upgrade
-from upgrade_tests.helpers.scenarios import create_dict
-from upgrade_tests.helpers.scenarios import get_entity_data
 
 from robottelo.constants import SYNC_INTERVAL
 from robottelo.datafactory import valid_cron_expressions
-from robottelo.test import APITestCase
 
 
-class ScenarioSyncPlan(APITestCase):
-    """The test class contains pre-upgrade and post-upgrade scenarios to test
+class TestSyncPlan:
+    """
+    The test class contains pre-upgrade and post-upgrade scenarios to test
```

Comment:
```suggestion
    The test class contains pre-upgrade and post-upgrade steps to test
```

Reply:
I think "scenario" is more appropriate than steps.

Reply:
When we combine Pre-Steps and Post_steps, it becomes `a Scenario`. PreUpgrade and PostUpgrade are not `scenarios` but steps of one test scenario :)

Reply:
Pre_upgrade + Post_upgrade --> Scenario not steps

Reply:
@devendra104 Ohh its that way then just do s/scenarios/scenario/.